apps/ledger-live-mobile/e2e/models/wallet/walletTabNavigator.ts

```diff
@@ -1,10 +1,9 @@
 import { getElementById, tapByElement } from "../../helpers";
-import { NavigatorName, ScreenName } from "../../../src/const";
+import { ScreenName } from "../../../src/const";
 
 export default class WalletTabNavigatorPage {
   nftGalleryTab = () => getElementById(`wallet-tab-${ScreenName.WalletNftGallery}`);
   portfolioTab = () => getElementById(`wallet-tab-${ScreenName.Portfolio}`);
-  marketTab = () => getElementById(`wallet-tab-${NavigatorName.Market}`);
```

Comment:
I think you could have just renamed the string in `getElementById` to `tab-bar-market` no?